controllers/rsyncnodedatamovement_controller.go

```diff
@@ -106,18 +106,6 @@ func (r *RsyncNodeDataMovementReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, err
 	}
 
-	if err := r.verifySourcePath(ctx, rsyncNode); err != nil {
```

Comment:
Are you going to get an error about an unused function?

Reply:
No but I'll remove it all the same - that does clean up the imports.